docs/rules/0192/trademarked-names.md

```diff
@@ -0,0 +1,88 @@
+---
+rule:
+  aip: 192
+  name: [core, '0192', trademarked-names]
+  summary: Trademarked names should be used correctly.
+permalink: /192/trademarked-names
+redirect_from:
+  - /0192/trademarked-names
+---
+
+# Trademarked names
+
+This rule enforces trademarked names in public comments are not abbreviated and
+follow the trademark owner's branding style, as mandated in [AIP-192][].
+
+## Details
+
+This rule looks at each descriptor in each proto file (exempting oneofs and the
```

Comment:
Why do we exempt oneofs ?

Reply:
We have not needed a oneof descriptor rule yet, and oneof documentation is unused to the best of my knowledge.